components/candy-machine/UpdateCandyMachine.tsx

```diff
@@ -313,7 +345,7 @@ const UpdateCandyMachine: FC<{
                                         required
                                     />
                                 </div>
-                                {errorMessage.length > 0 && (
+                                {error.length > 0 && (
```

Comment:
```suggestion
                                {error.?length > 0 && (
```
i know there is an if above, but it just makes the code more robust in case the if gets removed 🙏🏼